tools/pythonpkg/src/pyexpression/initialize.cpp

```diff
@@ -0,0 +1,246 @@
+#include "duckdb_python/pybind11/pybind_wrapper.hpp"
+#include "duckdb_python/expression/pyexpression.hpp"
+#include "duckdb/common/helper.hpp"
+#include "duckdb/common/vector.hpp"
+#include "duckdb_python/python_conversion.hpp"
+
+namespace duckdb {
+
+void InitializeStaticMethods(py::module_ &m) {
+	const char *docs;
+
+	// Constant Expression
+	docs = "Create a constant expression from the provided value";
+	m.def("ConstantExpression", &DuckDBPyExpression::ConstantExpression, py::arg("value"), docs);
+
+	// ColumnRef Expression
+	docs = "Create a column reference from the provided column name";
+	m.def("ColumnExpression", &DuckDBPyExpression::ColumnExpression, py::arg("name"), docs);
+
+	// Case Expression
+	docs = "";
+	m.def("CaseExpression", &DuckDBPyExpression::CaseExpression, py::arg("condition"), py::arg("value"), docs);
+
+	// Star Expression
+	docs = "";
+	m.def("StarExpression", &DuckDBPyExpression::StarExpression, py::kw_only(), py::arg("exclude") = py::list(), docs);
+	m.def(
+	    "StarExpression", []() { return DuckDBPyExpression::StarExpression(); }, docs);
+
+	// Function Expression
+	docs = "";
+	m.def("FunctionExpression", &DuckDBPyExpression::FunctionExpression, py::arg("function_name"), docs);
+}
+
+static void InitializeDunderMethods(py::class_<DuckDBPyExpression, shared_ptr<DuckDBPyExpression>> &m) {
+	const char *docs;
+
+	docs = R"(
+		Add two expressions.
+
+		Parameters:
+			expr: The expression to add together with
+
+		Returns:
+			FunctionExpression: A '+' on the two input expressions.
+	)";
+
+	m.def("__add__", &DuckDBPyExpression::Add, py::arg("expr"), docs);
+	m.def("__radd__", &DuckDBPyExpression::Add, py::arg("expr"), docs);
+
+	docs = R"(
+		TODO: add docs
+	)";
+	m.def("__neg__", &DuckDBPyExpression::Negate, docs);
+
+	docs = R"(
+		TODO: add docs
+	)";
+	m.def("__sub__", &DuckDBPyExpression::Subtract, docs);
+	m.def("__rsub__", &DuckDBPyExpression::Subtract, docs);
+
+	docs = R"(
+		TODO: add docs
+	)";
+	m.def("__mul__", &DuckDBPyExpression::Multiply, docs);
+	m.def("__rmul__", &DuckDBPyExpression::Multiply, docs);
+
+	docs = R"(
+		TODO: add docs
+	)";
+	m.def("__div__", &DuckDBPyExpression::Division, docs);
+	m.def("__rdiv__", &DuckDBPyExpression::Division, docs);
+
+	docs = R"(
+		TODO: add docs
+	)";
+	m.def("__truediv__", &DuckDBPyExpression::Division, docs);
+	m.def("__rtruediv__", &DuckDBPyExpression::Division, docs);
+
+	docs = R"(
+		TODO: add docs
+	)";
+	m.def("__floordiv__", &DuckDBPyExpression::FloorDivision, docs);
+	m.def("__rfloordiv__", &DuckDBPyExpression::FloorDivision, docs);
```

Comment:
what's the difference between `__floordiv__` and `__rfloordiv__`?

Reply:
In our case it doesn't matter because the only variant we accept is where `other` is also a class instance, so the `__rfloordiv__` variant likely doesn't even get used